docs/contributing/documentation_guidelines.md

```diff
@@ -0,0 +1,298 @@
+# Documentation guidelines
+
+```eval_rst
+.. contents::
+    :local:
+    :depth: 1
+```
+
+## Formatting recommendations
+
+Wagtail’s documentation uses a mixture of [Markdown](https://commonmark.org/help/) and [reStructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html). We encourage writing documentation in Markdown first, and only reaching for more advanced reStructuredText formatting if there is a compelling reason.
+
+Here are formats we encourage using when writing documentation for Wagtail.
+
+### Paragraphs
+
+It all starts here.
+Keep your sentences short, varied in length.
+
+Separate text with an empty line to create a new paragraph.
+
+### Heading levels
+
+Use heading levels to create sections, and allow users to link straight to a specific section. Start documents with an `# h1`, and proceed with `## h2` and further sub-sections without skipping levels.
+
+```md
+# Heading level 1
+
+## Heading level 2
+
+### Heading level 3
+```
+
+### Lists
+
+Use bullets for unordered lists, numbers when ordered. Prefer dashes `-` for bullets. Nest with indentation.
+
+```md
+- Bullet 1
+- Bullet 2
+    - Nested bullet 2
+- Bullet 3
+
+1. Numbered list 1
+2. Numbered list 2
+3. Numbered list 3
+```
+
+<details>
+
+<summary>Rendered output</summary>
+
+- Bullet 1
+- Bullet 2
+    - Nested bullet 2
+- Bullet 3
+
+1. Numbered list 1
+2. Numbered list 2
+3. Numbered list 3
+
+</details>
+
+### Inline styles
+
+Use **bold** and _italic_ sparringly, inline `code` when relevant.
+
+```md
+Use **bold** and _italic_ sparringly, inline `code` when relevant.
+```
+
+### Code blocks
+
+Make sure to include the correct language code for syntax highlighting, and to format your code according to our styleguides. Frequently used: `python`, `css`, `html`, `html+django`, `javascript`, `console`.
```

Comment:
```suggestion
Make sure to include the correct language code for syntax highlighting, and to format your code according to our coding guidelines. Frequently used: `python`, `css`, `html`, `html+django`, `javascript`, `console`.
```

Maybe "coding guidelines" since that's what readers will see in the sidebar to the left?